README.md

```diff
@@ -8,23 +8,57 @@ The DATS project will be transferring inactive Full Retention (FR) government di
 
 ## Quick Start
 
-1. Set up the `.env` based on the `.env.template` file.
+1. **Set Up Environment**: Set up the `.env` based on the `.env.template` file.
 
-2. Run `npm run up` to start the API, MongoDB, and RabbitMQ services.
+2. **Start Services**: Run `npm run up` to start the API, MongoDB, and RabbitMQ services.
 
-3. Change directory to `desktop` and run `npm run dev` to launch the desktop app.
+3. **Desktop Directory**: Navigate to `desktop`.
+
+4. **Install Dependencies**: Run `npm install`.
+
+5. **Build App**: Run `npm run build` to enable worker functionality.
```

Comment:
add note on running as admin based on os?

Reply:
Great idea!

Reply:
Actually now that I think about it I'm not sure if it's necessary. On Windows I can run `npm run build` without Admin privs, but not `npm run build:win`.

`npm run build` just creates the `out` directory and not the `dist`. Is it the same on Mac?